R/get_error_path.R

```diff
@@ -73,17 +77,30 @@ get_error_path <- function(schema, element = "target_metadata", # nolint: cycloc
     unique() |>
     detect_first()
 
-
   # Instance paths to custom task IDs (which will not have been matched in the schema)
   # can be created by appending the element name to the task ID properties path.
   if (length(path) == 0L && type == "instance") {
-    path <- grep(paste0(".*", "task_ids", "/type([0-9])?$"),
+    task_ids_paths <- grep(
+      paste0(".*", "/task_ids", "/type([0-9])?$"),
       schema_paths,
       value = TRUE
     ) %>%
       gsub("/type([0-9])?", "", .) %>%
-      unique() %>%
-      paste("properties", element, sep = "/")
+      unique()
+
+    # Error if we found multiple paths (indicates regex bug from missing leading /)
+    if (length(task_ids_paths) > 1L) {
+      cli::cli_abort(
+        c(
+          "Multiple task_ids base paths found in schema for custom task ID {.val {element}}.",
```

Comment:
what is a base path? This error message would not be clear to me as a hub admin.